### DISCUSSION TITLE
broadcast read permission issue

### DISCUSSION THREAD

**Discussion by chanduittamsetty** [Cyfrin/foundry-full-course-cu#2091] [Q&A]
<img width="1110" alt="Screenshot 2024-07-23 at 5 18 49 PM" src="https://github.com/user-attachments/assets/02c64412-73c2-4339-bb49-75c1129260d9">
I am facing the broadcast read issue even after adding permissions in tomil 

fs_permissions = [
    { access = "read", path = "./broadcast" },
    { access = "read", path = "./reports" },
]

command used 

forge test --mt testUserCanFundInteractions --rpc-url $RPC_URL -vvv

**Comment by EngrPips** [comment]
Hello @chanduittamsetty, Please show your `foundry.toml` file and also run `foundryup,` then retry. Let's see if the issue gets resolved.
